### PR TITLE
Develop vova -> latest

### PR DESCRIPTION
**Pull request by planv**
- fixed JS errors that block creating keys
- added new GitHub Action to create SBOM and release

### CODE REVIEW

File: .github/workflows/publish_release.yml [hunk 1]
```diff
@@ -0,0 +1,130 @@
+name: Publish Release
+run-name: Publish Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: 'Release version (e.g. 1.2.3)'
```
Comment (kavaribes) [inline .github/workflows/publish_release.yml:8]: @planv this should probably be more informative

**Review by kavaribes — APPROVED** [review]

File: .github/workflows/publish_release.yml (hunk 1 above)
Comment (planv) [reply .github/workflows/publish_release.yml:8]: Something like `Please type release version (e.g. 1.2.3)` ?

Comment (fqjony) [reply .github/workflows/publish_release.yml:8]: @planv I would recommend checking out https://github.com/udx/udx-worker/blob/latest/.github/workflows/build.yml 

I used `gittools/actions/gitversion/execute@v0` for that purpose. I'm not sure if it's what you need, though.

**Comment by fqjony** [comment]
@planv, so, do we need the temp branch and draft release to attach the sbom artifact, or is there any other reason to do it that way? Just for my understanding.

**Review by fqjony — APPROVED** [review]

**Comment by planv** [comment]
> 
@planv, so, do we need the temp branch and draft release to attach the sbom artifact, or is there any other reason to do it that way? Just for my understanding.


Yes, and also for cleaning up files that should not be included to release.


But it looks like it is an old version, and I have already improved it here - https://github.com/udx/docker-site/blob/latest/.github/workflows/build-and-release.yml

We have to implement 1 process that we will use to deploy our Docker Images - Docker Site, Docker SFTP, Rabbit Updater etc

**Comment by planv** [comment]
@fqjony @kavaribes I removed GitHub Action. It will be refactored and then I'll commit it.
Let's release fixes for now

File: .github/workflows/publish_release.yml (hunk 1 above)
Comment (planv) [reply .github/workflows/publish_release.yml]: Will be reviewed for the next version, closing for now

**Review by fqjony — APPROVED** [review]

**Pull request merged**